content/en/checkout/migration.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Migration
+description: ""
+position: 34
+category: Checkout
+---
+
```

Comment:
Should we add how to install with npm?

Reply:
For a customer that wants to integrate like that. For those more "advanced" customers that are using frameworks and stuff